pegasus/sites.v3/code.org/public/students.haml

```diff
@@ -3,18 +3,20 @@ title: Learn Computer Science
 theme: responsive_full_width
 ---
 
+-# Determine if a user is signed in
+- cookie_key = environment_specific_cookie_name("_user_type")
+- user_type = request.cookies[cookie_key]
```

Comment:
Just checking -- have you tested this locally? I was looking at a different `userType` thing recently and the user type for signed out users was "unknown". That was in our redux store instead of in the cookie though, so definitely doesn't mean that the same behavior applies here!

Reply:
Yep! I tested locally, and it worked as expected.